Gemfile

```diff
@@ -14,4 +14,8 @@ end
 # your gem to rubygems.org.
 
 # Load Gemfile with dependencies from manageiq
-eval_gemfile(File.expand_path("spec/manageiq/Gemfile", __dir__))
+MANAGEIQ_GEMFILE = File.expand_path("spec/manageiq/Gemfile", __dir__)
+unless File.exist?(MANAGEIQ_GEMFILE)
+  raise 'You must run `bin/setup` to install manageiq into spec/manageiq as a rails dummy app for this provider engine gem'
```

Comment:
We had a similar problem in manageiq-content and solved it similarly.  We should just make them all consistent.  See  https://github.com/ManageIQ/manageiq-content/blob/master/Gemfile#L5-L13